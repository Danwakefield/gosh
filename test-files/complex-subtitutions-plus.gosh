EMPTY=
FULL=SUCCESS
echo ${NOTSET+SUCCESS}
echo ${EMPTY:+SUCCESS}
echo "Next Two Lines Should Be Empty"
echo ${FULL+FAIL}
echo ${FULL:+FAIL}
