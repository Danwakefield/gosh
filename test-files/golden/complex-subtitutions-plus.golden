SUCCESS
SUCCESS
Next Two Lines Should Be Empty


